src/ducks/apps/components/UninstallModal.jsx

```diff
@@ -40,6 +40,11 @@ export class UninstallModal extends Component {
 
   render() {
     const { isUninstalling, dismissAction, t, uninstallError } = this.props
+    const linkedAppError =
+      uninstallError &&
+      uninstallError.message === 'A linked OAuth client exists for this app'
+    const settingsAppURL =
+      '//' + window.location.host.replace('store', 'settings')
```

Comment:
I'm not proud of this hack

Reply:
I think we need a redirection intent to settings here, redirection intent was made exactly for this need